tools/c7n_mailer/c7n_mailer/cli.py

```diff
@@ -35,6 +35,10 @@
         'ldap_uri': {'type': 'string'},
         'ldap_bind_dn': {'type': 'string'},
         'ldap_bind_user': {'type': 'string'},
+        'ldap_username_attribute': {'type': 'string'},
+        'ldap_manager_attribute': {'type': 'string'},
```

Comment:
If folks have different attributes, like uid instead of samaccountname they can set those for their specific AD schema